src/main/java/org/jabref/gui/groups/GroupTreeController.java

```diff
@@ -43,14 +43,21 @@
 
     private static final Log LOGGER = LogFactory.getLog(GroupTreeController.class);
 
-    @FXML private TreeTableView<GroupNodeViewModel> groupTree;
-    @FXML private TreeTableColumn<GroupNodeViewModel, GroupNodeViewModel> mainColumn;
-    @FXML private TreeTableColumn<GroupNodeViewModel, GroupNodeViewModel> numberColumn;
-    @FXML private TreeTableColumn<GroupNodeViewModel, GroupNodeViewModel> disclosureNodeColumn;
-    @FXML private CustomTextField searchField;
+    @FXML
+    private TreeTableView<GroupNodeViewModel> groupTree;
```

Comment:
Please leave the formatting as it is right now. It is kind of a JavaFX-thing that normally the @FXML tag is on the same line (and I like it that way because otherwise the private fields declaration takes more space)